File: src/practice/luogu/p5708/Main.py
# https://www.luogu.com.cn/problem/P5708
import math
(a, b, c) = list(map(float, input().split()))
p = (a + b + c) / 2
print('%.1f' % math.sqrt(p * (p - a) * (p - b) * (p - c)))

File: src/practice/luogu/p5708/Main.php
<?php
// https://www.luogu.com.cn/problem/P5708
fscanf(STDIN, '%f%f%f', $a, $b, $c);
$p = ($a + $b + $c) / 2;
echo sprintf('%.1f', sqrt($p * ($p - $a) * ($p - $b) * ($p - $c)));
